app/services/albums.js

```diff
@@ -0,0 +1,18 @@
+const fetch = require('node-fetch');
+
+const errors = require('../errors');
+
+exports.getAlbums = () =>
+  fetch('https://jsonplaceholder.typicode.com/albums')
```

Comment:
Use a variable `env` for` https: // jsonplaceholder.typicode.com` because it must be configurable and maintainable